demes/demes.py

```diff
@@ -2103,7 +2103,8 @@ class Builder:
 
     :ivar dict data: The data dictionary of the graph's current state.
         The objects nested within this dictionary follow Demes' data model,
-        as described in the :ref:`schema <sec_schema>`.
+        as described in the
+        `schema <https://popsim-consortium.github.io/demes-spec-docs/main/spec.html>`_.
```

Comment:
Better to use intersphinx for this - it's a sphinx site, right, so it has an objects.inv etc.

Reply:
Oh cool, thanks. I've made about 100 tiny mistakes here though... maybe I should just quit and go have a beer instead.

Reply:
If you have to ask the question, it's definitely time for beer. Enjoy!